src/main/resources/duelistModResources/localization/eng/DuelistMod-Card-Strings.json

```diff
@@ -10757,5 +10757,92 @@
+	},
+	"theDuelist:GustoEgul":  {
+		"DESCRIPTION": "Gusto Beast NL Summon !duelist:SUMM! NL Gain !B! Block. NL When you Tribute this monster for a Spellcaster, Fetch a Spellcaster from your draw pile.",
+		"NAME": "Gusto Egul",
+		"UPGRADE_DESCRIPTION": "Gusto Beast NL Summon !duelist:SUMM! NL Gain !B! Block. NL When you Tribute this monster for a Spellcaster, Fetch a Spellcaster from your draw pile. Upgrade it."
+	},
+	"theDuelist:GustoWindaar": {
+		"DESCRIPTION": "Gusto Spellcaster NL Tribute 2 NL Deal !D! damage. NL Gain !B! Block. NL Upgrade a random Beast in your discard pile.",
+		"NAME": "Sage of Gusto",
+		"UPGRADE_DESCRIPTION":  "Gusto Spellcaster NL Tribute 2 NL Deal !D! damage. NL Gain !B! Block. NL Upgrade ALL Beasts in your discard pile."
+	},
+	"theDuelist:ContactWithGusto": {
+		"DESCRIPTION": "Shuffle ALL Spellcaster or Beast monsters from your discard pile into your draw pile, gain !M! *Plated *Armor for each. NL Exhaust",
+		"NAME": "Contact with Gusto",
+		"UPGRADE_DESCRIPTION": "Shuffle ALL Spellcaster or Beast monsters from your discard pile into your draw pile, gain !M! *Plated *Armor for each. NL Exhaust"
+	},
+	"theDuelist:GustoBlessings": {
+		"DESCRIPTION": "Fetch 1 of 3 random Spellcaster or Beast monsters in your discard pile, shuffle the rest into your draw pile. NL Draw a card. NL Exhaust",
+		"NAME": "Blessings for Gusto",
+		"UPGRADE_DESCRIPTION": "Fetch an Upgraded copy of 1 of 3 random Spellcaster or Beast monsters in your discard pile, shuffle the rest into your draw pile. NL Draw a card."
+	},
+	"theDuelist:GustoPilica": {
+		"DESCRIPTION": "Gusto Spellcaster NL Summon !duelist:SUMM! NL Shuffle a random Gusto Beast into your draw pile. NL Channel an Air. NL Exhaust",
+		"NAME": "Descendant of Gusto",
+		"UPGRADE_DESCRIPTION": "Gusto Spellcaster NL Summon !duelist:SUMM! NL Shuffle two random Upgraded Gusto Beasts into your draw and discard piles. NL Channel an Air. NL Exhaust"
+	},
+	"theDuelist:GustoSquirro": {
+		"DESCRIPTION": "Gusto Beast NL Summon !duelist:SUMM! NL Deal !D! damage. NL Gain !B! Block. NL When you Tribute this monster for a Spellcaster, Scry !M!.",
+		"NAME": "Gusto Squirro",
+		"UPGRADE_DESCRIPTION": "Gusto Beast NL Summon !duelist:SUMM! NL Deal !D! damage. NL Gain !B! Block. NL When you Tribute this monster, Scry !M!."
+	},
+	"theDuelist:GustoGulldo": {
+		"DESCRIPTION": "Gusto Beast NL Summon !duelist:SUMM! NL Deal !D! damage. NL When you Tribute this monster, shuffle a random Gusto Spellcaster into your draw pile. NL Exhaust",
+		"NAME": "Gusto Gulldo",
+		"UPGRADE_DESCRIPTION": "Gusto Beast NL Summon !duelist:SUMM! NL Deal !D! damage. NL When you Tribute this monster, shuffle a random Upgraded Gusto Spellcaster into your draw pile. NL Exhaust"
+	},
+	"theDuelist:GustoFalco": {
+		"DESCRIPTION": "Gusto Beast NL Summon !duelist:SUMM! NL Deal !D! damage to ALL enemies !M! times. NL When you Tribute this monster, Resummon a Spellcaster from your discard pile.",
```

Comment:
Not using Resummon keyword anymore. Should be one of the following references in the card JSON, see OniTankT34

- ~SpecialS~~ (Special Summon)
- ~~SpecialS'ed~~ (Special Summoned)
- ~~SpecialS'ing~~ (Special Summoning)
- ~~SpecialS's~~ (Special Summons)